app/connectors/EoriDetailsConnector.scala

```diff
@@ -17,22 +17,22 @@
 package connectors
 
 import config.AppConfig
+import connectors.httpParsers.EoriDetailsHttpParser.EoriDetailsReads
 import connectors.httpParsers.ResponseHttpParser.HttpGetResult
-import connectors.httpParsers.ImporterAddressHttpParser.TraderAddressReads
-import models.TraderAddress
+import models.EoriDetails
 import uk.gov.hmrc.http.{HeaderCarrier, HttpClient}
 
 import javax.inject.{Inject, Singleton}
 import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
-class ImporterAddressConnector @Inject()(val http: HttpClient,
-                                         implicit val config: AppConfig){
+class EoriDetailsConnector @Inject()(val http: HttpClient,
+                                     implicit val config: AppConfig) {
 
-  private[connectors] def getAddressUrl(id: String) = s"${config.sub09}/api/address/$id"
+  private[connectors] def getEoriDetailsUrl(id: String) = s"${config.sub09}/api/eoriDetails/$id"
```

Comment:
To be in line with the EIS endpoint please update this to include the proper query string parameters.